java-demo/Dockerfile

```
@@ -0,0 +1,38 @@
+FROM eclipse-temurin:21-jdk-alpine AS builder
+
+WORKDIR /build
+
+COPY ./.mvn ./.mvn
+COPY ./mvnw ./mvnw
+COPY ./pom.xml ./pom.xml
+COPY ./src ./src
+
+RUN ./mvnw clean package
+
+FROM eclipse-temurin:21-jre-alpine AS runtime
+
+# Common base image.
+LABEL org.opencontainers.image.authors="matcaz@noc.ac.uk"
```

I don't think I should be listed as the author for this one! 😄 
```suggestion
LABEL org.opencontainers.image.authors="Leo.Bruvry.Lagadec@ifremer.fr"
```